OstelcoStyles/LoopingVideoView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LoopingVideoView.swift
+//  OstelcoStyles
+//
+//  Created by Ellen Shapiro on 5/22/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+/// A view designed to play looping videos to look like GIFs
+/// without the horrendous memory overhead of actual GIFs
+@IBDesignable
+public class LoopingVideoView: UIView {
+    
+    private lazy var gifPlayer = AVQueuePlayer()
```

Comment:
Are we play GIF or MP4s ? If its MP4 change the name

Reply:
we are playing mp4s of gifs 🤷‍♀

I suppose `loopingVideoPlayer` would probably be more accurate though